fastai/dataset.py

```diff
@@ -82,8 +82,18 @@ def n_hot(ids, c):
     return res
 
 def folder_source(path, folder):
+  """
+  Returns the filenames and labels for a folder within a path
+  
+  Returns:
+  -------
+  fnames: a list of the filenames within `folder`
+  all_lbls: a list of all of the labels in `folder`, where the # of labels is determined by the # of directories within `folder`
+  lbl_arr: a numpy array of the label indices in `all_lbls`
+  
+  """
     fnames, lbls, all_lbls = read_dirs(path, folder)
-    lbl2idx = {v:k for k,v in enumerate(all_lbls)}
```

Comment:
I don't mind this change - but FYI I pretty much always use `k,v` for key and value in enumerations and dict comprehensions. Sorry if it's missing from the abbrev file!

Reply:
oh i see my bad, i'll remember that 👍I was thinking that `k` meant `key` for objects specifically

And don't worry `key` and `value` are in the abbrev file!